app/_data/docs_nav_gateway_3.0.x.yml

```diff
@@ -11,504 +11,529 @@ items:
+        url: /admin-api/
+      - text: Information Routes
+        url: /admin-api/#information-routes      
+      - text: Health Routes
+        url: /admin-api/#health-routes
+      - text: Tags
+        url: /admin-api/#tags
+      - text: Service Object
+        url: /admin-api/#service-object
+      - text: Route Object
+        url: /admin-api/#route-object
+      - text: Consumer Object
+        url: /admin-api/#consumer-object
+      - text: Plugin Object
+        url: /admin-api/#plugin-object
+      - text: Certificate Object
+        url: /admin-api/#certificate-object
+      - text: CA Certificate Object
+        url: /admin-api/#ca-certificate-object
+      - text: SNI Object
+        url: /admin-api/#sni-object
+      - text: Upstream Object
+        url: /admin-api/#upstream-object
+      - text: Target Object
+        url: /admin-api/#target-object
+      - text: Vaults Beta
+        url: /admin-api/#vaults-beta-entity
+      - text: Licenses
+        url: /admin-api/licenses/reference
+        items:
+          - text: Licenses Reference
+            url: /admin-api/licenses/reference
+          - text: Licenses Examples
+            url: /admin-api/licenses/examples
+      - text: Workspaces
+        url: /admin-api/workspaces/reference
+        items:
+          - text: Workspaces Reference
+            url: /admin-api/workspaces/reference
+          - text: Workspace Examples
+            url: /admin-api/workspaces/examples
+      - text: RBAC
+        url: /admin-api/rbac/reference
+        items:
+          - text: RBAC Reference
+            url: /admin-api/rbac/reference
+          - text: RBAC Examples
+            url: /admin-api/rbac/examples
+      - text: Admins
+        url: /admin-api/admins/reference
+        items:
+          - text: API Reference
+            url: /admin-api/admins/reference
+          - text: Examples
+            url: /admin-api/admins/examples
+      - text: Developers
+        url: /admin-api/developers/reference
+      - text: Consumer Groups
+        url: /admin-api/consumer-groups/reference
+        items:
+          - text: API Reference
+            url: /admin-api/consumer-groups/reference
+          - text: Examples
+            url: /admin-api/consumer-groups/examples
+      - text: Event Hooks
+        url: /admin-api/event-hooks/reference
+        items:
+          - text: Event Hooks Reference
+            url: /admin-api/event-hooks/reference
+          - text: Examples
+            url: /admin-api/event-hooks/examples
+      - text: Securing the Admin API
+        url: /admin-api/secure-admin-api
+  - title: Reference
+    icon: /assets/images/icons/documentation/icn-references-color.svg
+    items:
+      - text: Configuration Reference
+        url: /reference/configuration
+      - text: Kong Gateway Nginx Directives
+        url: /reference/nginx-directives
+      - text: CLI Reference
+        url: /reference/cli
+      - text: Rate Limiting Library
```

Comment:
This is pointing to the same page as "Configuring sliding-window". Not sure if that's intentional.

Reply:
I did do it intentionally as a placeholder, but I'll fix it here.